docker/HOW_TO_USE_DOCKER.md

```diff
@@ -89,13 +91,56 @@ mintplexlabs/anythingllm;
 Go to `http://localhost:3001` and you are now using AnythingLLM! All your data and progress will persist between
 container rebuilds or pulls from Docker Hub.
 
+> [!TIP]
```

Comment:
If you look at the bottom there is already a `Common questions and fixes` section and this should go there.

It also might be worth to have all Q/A questions be a <details><summary> HTML embed in the Markdown to make the document not so long.

Or maybe we add a README.md into the `aiProviders/ollama` folder that has this exact info and links to it from some link on this page in the QA section.